docs/deployment/pull-backup.rst

```diff
@@ -0,0 +1,92 @@
+.. include:: ../global.rst.inc
+.. highlight:: none
+
+Backing up in pull mode
+=======================
+
+Assume you have a pull backup system set up with borg, where a backup server
+pulls the data from the target via sshfs.
+
+In this mode, the backup client's file system is mounted via sshfs on the backup
+server. Pull mode is even possible if the SSH connection must be established by
+the client via remote tunnel.
+
+Creating backup
+---------------
+
+In this approach the client file system is simply mounted and then backed up.
+
+::
+
+    mkdir /mnt/sshfs
+    sshfs root@TARGET_IP:/ /mnt/sshfs
+    borg create /REPOSITORY::archive /mnt/sshfs
+
+Note that this will back up every file with the ``/mnt/sshfs`` prefix. You can
+get rid of this prefix by changing the working directory before creating the
+backup:
+
+::
+
+    cd /mnt/sshfs
+    borg create /REPOSITORY::archive
+
+Restoring a full backup
+-----------------------
+
+In case of a restore there is no direct way for borg to know the correct UID:GID
+of files, because it uses the ``/etc/passwd`` and ``/etc/group`` of the backup
+server. Using the archived UIDs/GIDs might lead to wrong results, because after
+a reinstall the name-to-ID mapping might have changed.
+
+The workaround is chrooting into an sshfs mounted directory.
+
+::
+
+    mkdir /tmp/sshfs
+    sshfs root@TARGET_IP:/ /tmp/sshfs
+    mkdir /tmp/sshfs/borgrepo
+    mount --bind /REPOSITORY /tmp/sshfs/borgrepo
+    cp /usr/local/bin/borg /tmp/sshfs/usr/local/bin/borg
```

Comment:
this might only work for the borg fat binary - if one is using this from that location.

Reply:
I'll add a sentence about the stand-alone binary used here for example purposes.